Sources/Tracking/CustomerIO.swift

```diff
@@ -326,4 +335,25 @@ public class CustomerIO: CustomerIOInstance {
 
         implementation?.screen(name: name, data: data, jsonEncoder: jsonEncoder)
     }
+
+    public func automaticScreenView(
+        name: String,
+        data: [String: Any]
+    ) {
+        automaticScreenView(name: name, data: StringAnyEncodable(data))
+    }
+
+    // Designed to be called from swizzled methods for automatic screen tracking.
```

Comment:
does this mean that in RH we'll end up continuing to track screenviews in other workspaces when the siteId changes? or do we trash the siteId from the datastore when switching?

Reply:
> does this mean that in RH we'll end up continuing to track screenviews in other workspaces when the siteId changes?

Yes it will. 

The problem is that swizzled methods have no memory of what siteId to use. I can't think of any way to solve the problem?

Reply:
we could just flag the most recent siteID in the datastore and decide that auto screen views are only evaluated for the most recently enabled siteId and none others. If someone in future has a use case where they legitimately need CIO creds for two workspaces active for this feature at once, we can re-eval then

Reply:
> If someone in future has a use case where they legitimately need CIO creds for two workspaces active for this feature at once, we can re-eval then

We could keep track of *active* workspaces in the SDK. When the SDK initializes when the app has a cold start, *active* SDKs is empty. Then, every time that an instance of CIO is iniaitlized, we keep track at runtime of *active* siteIds. 

So, we will will *not* report automatic screen view events to workspaces that have not been used in a while.